### PR TITLE
connect all poker logic

### PR DESCRIPTION
**Pull request by makandrii**

### CODE REVIEW

**Review by harpestes — CHANGES_REQUESTED** [review]
Add data validation into request records

File: src/main/java/com/vamae/poker/model/responses/StartResponse.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.vamae.poker.model.responses;
+
+public record StartResponse(
```
Comment (harpestes) [inline src/main/java/com/vamae/poker/model/responses/StartResponse.java]: Unused record

**Review by harpestes — CHANGES_REQUESTED** [review]
Add data validation in request records

**Review by v1adem — APPROVED** [review]
Good work man

**Review by harpestes — APPROVED** [review]

**Pull request merged**